VirtoCommerce.ImageToolsModule.Web/Scripts/blades/task-list.tpl.html

```diff
@@ -15,13 +15,13 @@
             <div class="table-wrapper" ng-if="items.length > 0">
                 <div ui-grid="gridOptions" ui-grid-auto-resize ui-grid-save-state ui-grid-selection ui-grid-resize-columns ui-grid-move-columns ui-grid-pinning ui-grid-height ui-grid-infinite-scroll></div>
                 <ul class="menu __context" role="menu" id="cil_menu_{{blade.level}}">
-                    <li class="menu-item" ng-click='edit(contextMenuEntity)'>
+                    <li class="menu-item" ng-click='taskEdit(contextMenuEntity)'>
                         <i class="menu-ico fa fa-edit"></i> {{'platform.commands.manage' | translate}}
                     </li>
-                    <li class="menu-item" ng-click='cut(contextMenuEntity)' va-permission="catalog:create">
+                    <li class="menu-item" ng-click='taskRun(contextMenuEntity)' va-permission="catalog:create">
                         <i class="menu-ico fa fa-exclamation"></i> {{'imageTools.commands.run' | translate}}
                     </li>
-                    <li class="menu-item" ng-click='delete(contextMenuEntity)' va-permission="catalog:delete">
+                    <li class="menu-item" ng-click='taskDelete(contextMenuEntity)' va-permission="catalog:delete">
```

Comment:
Could you also use proper permission for the buttons - e.g. instead va-permission="catalog:delete" it should be "thumbnail:delete", and so on. Check appropriate permissions in API methods, please